server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -95,7 +95,17 @@
 import com.google.common.base.Preconditions;
 
 /**
- * Handles upgrading from 2.0 to 2.1
+ * Handles upgrading from 2.0 to 2.1.
```

Comment:
@milleruntime does this cover what you were looking for?

Reply:
This is great, thanks. I was also thinking we could link the methods associated with the change but that might not always be easy

Reply:
I added a reference to the associated upgrade method.